internal/cloudapi/v2/compose.go

```diff
@@ -35,18 +35,18 @@ func (bcpm BlueprintCustomizationsPartitioningMode) String() string {
 	}
 }
 
-// GetCustomizationsFromBlueprintRequest populates a blueprint customization struct
-// with the data from the blueprint section of a ComposeRequest, which is similar but
+// GetCustomizationsFromBlueprint populates a blueprint customization struct
+// with the data from request Blueprint, which is similar but
 // slightly different from the Cloudapi's Customizations section
 // This starts with a new empty blueprint.Customization object
 // If there are no customizations, it returns nil
-func (request *ComposeRequest) GetCustomizationsFromBlueprintRequest() (*blueprint.Customizations, error) {
-	if request.Blueprint.Customizations == nil {
+func (rbp *Blueprint) GetCustomizationsFromBlueprintRequest() (*blueprint.Customizations, error) {
```

Comment:
The function name should be `GetCustomizationsFromBlueprint`, judging from the comment above it.